src/WordPress.php

```diff
@@ -406,9 +414,15 @@ private function maybe_verify_remote_db_connection() {
 				throw new \Exception( 'There was a problem inspecting network. Please check the logs' );
 			}
 		}
-		\EE::log( 'Verifying connection to remote database' );
+		if ( GLOBAL_DB !== $this->site_data['db_host'] ) {
+			\EE::log( 'Verifying connection to remote database' );
+		}
+		$img_versions = \EE\Utils\get_image_versions();
+		if ( GLOBAL_DB === $this->site_data['db_host'] ) {
+			$network = ( GLOBAL_DB === $this->site_data['db_host'] ) ? "--network='" . GLOBAL_NETWORK . "'" : '';
+		}
 
-		if ( ! \EE::exec( sprintf( "docker run -it --rm --network='%s' mysql sh -c \"mysql --host='%s' --port='%s' --user='%s' --password='%s' --execute='EXIT'\"", $this->site_data['site_url'], $this->site_data['db_host'], $this->site_data['db_port'], $this->site_data['db_user'], $this->site_data['db_password'] ) ) ) {
+		if ( ! \EE::exec( sprintf( "docker run -it --rm %s easyengine/mariadb:%s sh -c \"mysql --host='%s' --port='%s' --user='%s' --password='%s' --execute='EXIT'\"", $network, $img_versions['easyengine/mariadb'], $this->site_data['db_host'], $this->site_data['db_port'], $this->site_data['db_user'], $this->site_data['db_password'] ) ) ) {
```

Comment:
Here `$network` variable will not be initialised if `GLOBAL_DB !== $this->site_data['db_host']`

Reply:
Yes. In that case network is not needed, empty string is needed and that is the case.